resources/codegen_inputs/layout.rs

```diff
@@ -611,14 +611,17 @@ table ConditionSet {
     /// Array of offsets to condition tables, from beginning of the
     /// ConditionSet table.
     #[count($condition_count)]
-    condition_offsets: [Offset32<ConditionFormat1>],
+    condition_offsets: [Offset32<Condition>],
 }
 
-///// [Condition Table](https://docs.microsoft.com/en-us/typography/opentype/spec/chapter2#condition-table)
-//Condition {
-    ///// FIXME: make an enum
-    //no_field: fake,
-//}
+/// [Condition Table](https://docs.microsoft.com/en-us/typography/opentype/spec/chapter2#condition-table)
+format u16 Condition {
```

Comment:
I would expand on the docs a bit to mention that we are including items that are not yet part of the spec, and then link to whatever the best current reference is?

Reply:
Good idea, done